ext/libclementine-tagreader/tagreader.cpp

```diff
@@ -128,8 +128,45 @@ namespace {
 const char* kMP4_OriginalYear_ID = "----:com.apple.iTunes:ORIGINAL YEAR";
 const char* kASF_OriginalDate_ID = "WM/OriginalReleaseTime";
 const char* kASF_OriginalYear_ID = "WM/OriginalReleaseYear";
+
+// Helpers for GuessArtistAndTitle()
+QString WithoutExtension(const QString& s) {
+  if (s.isEmpty()) return s;
+  const int i = s.lastIndexOf('.');
+  if (i  < 0) return s;
+  return s.left(i);
+}
+
+void ReplaceUnderscoresWithSpaces(QString &s) {
```

Comment:
You can make this `const` if you make an explicit copy, i.e.,
```c++
QString ret(s);
ret.replace(...);
return ret;
```

Reply:
Good point, done